controllers/front/CartController.php

```diff
@@ -637,14 +648,14 @@ protected function areProductsAvailable()
         if ($product['active']) {
             return $this->trans(
                 'The item %product% in your cart is no longer available in this quantity. You cannot proceed with your order until the quantity is adjusted.',
```

Comment:
Wouldn't it be possible to replace "item" with "product" here for more consistency?

Reply:
Done